checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -70,6 +70,8 @@ def analyze_line(
             line_number=line_number,
             context=raw_context,
             is_multiline=False,
+            is_added=is_added,
+            is_removed=is_removed,
```

Comment:
@lirshindalman Why does our custom detector need to know this? Can't it be abstracted away in the runner only?